sonic_data_client/db_client.go

```diff
@@ -609,6 +609,10 @@ func populateDbtablePath(prefix, path *gnmipb.Path, pathG2S *map[*gnmipb.Path][]
 		if err != nil {
 			log.Errorf("Could not create CountersPfcwdNameMap: %v", err)
 		}
+		err = initCountersFabricPortNameMap()
+		if err != nil {
+			return err
```

Comment:
Perhaps we can log errorf instead of returning error, if map is unable to be created. If we are unable to create this map all COUNTERS_DB queries will fail.

Reply:
Fixed as suggested.